player_rating/tests.py

```diff
@@ -0,0 +1,103 @@
+from .models import PlayerRating, Rating
+from player.models import BallGame, Player
+from django.core.exceptions import ValidationError
+from django.db import IntegrityError
+from django.contrib.auth import get_user_model
+import pytest
+
+
+@pytest.mark.django_db
+class TestPlayerModel:
+
+    @pytest.fixture
+    def user_fixture(self):
+        return get_user_model().objects.create_user(
+            username='testuser',
+            password='testpass'
+        )
+
+    @pytest.fixture
+    def player_fixture(self, user_fixture):
+        player = Player.objects.create(
+            user=user_fixture,
+            birth_date='1990-01-01',
+            favorite_ball_game=BallGame.Soccer
+        )
+        return player
+
+    @pytest.fixture
+    def player_rating_fixture(self, player_fixture):
+        player_rating_fixture = PlayerRating.objects.create(
+            ball_game=BallGame.Soccer,
+            user_id=player_fixture,
+            rating=Rating.Five
+        )
+        return player_rating_fixture
+
+    def test_get_player_rating(self, player_rating_fixture):
+        player_rating_from_db = PlayerRating.objects.get(
+            pk=player_rating_fixture.pk)
+        assert player_rating_from_db == player_rating_fixture
+
+    def test_update_player_rating(self, player_rating_fixture):
+        player_rating_fixture.rating = Rating.Seven
```

Comment:
Add an assertion that it was not seven before the change to make the test extra clear and safe.

Reply:
Done.